src/github/middleware.ts

```diff
@@ -173,6 +167,12 @@ export default (
 			context.log = context.log.child({ jiraHost });
 			context.log("Processing event for Jira Host");
 
+			// Metrics for webhook payload size
+			if (await booleanFlag(BooleanFlags.PAYLOAD_SIZE_METRIC, false, jiraHost)) {
+				emitWebhookPayloadMetrics(webhookEvent,
+					Buffer.byteLength(JSON.stringify(context.payload), "utf-8"));
+			}
+
```

Comment:
If we put it here the metric won't be accurate anymore, because it will be published per each Jira Instance subscription. So it could be multiple times per webhook.

What if we'll use this flag without Jira Host? We don't really need the progressive rollout here, so we don't really need Jira Host.

Reply:
Thats correct. I will remove the `jiraHost` value and move it outside to where it was.